webmails/snuffleupagus.rules

```diff
@@ -84,7 +84,7 @@ sp.disable_function.function("ini_set").param("option").value("include_path").dr
 sp.disable_function.function("ini_set").param("option").value("open_basedir").drop();
 
 # Detect some backdoors via environment recon
-sp.disable_function.function("ini_get").param("option").value("allow_url_fopen").drop();
+# sp.disable_function.function("ini_get").param("option").value("allow_url_fopen").drop(); # Disabled as this is necessary for carddav plugin to work
```

Comment:
Please enable it only for the required file instead of disabling it globally.
https://snuffleupagus.readthedocs.io/config.html#virtual-patching

Reply:
Done ;)